api/routes/officerManager.js

```diff
@@ -72,6 +80,13 @@ router.post('/delete', (req, res) => {
   Manager.deleteOne({ email: req.body.email }, function(error, form) {
     if (error) {
       logger.log(`Officer Manager /Delete3DForm error: ${error}`);
+      const info = {
+        userEmail:req.body.email,
```

Comment:
put a space in between the colon and req.body so it looks like
`userEmail: req.body.email,`